tutorials/milvus_llamaindex_search_and_retrieval_tutorial.ipynb

```diff
@@ -0,0 +1,751 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    !pip install https://github.com/filip-halt/llama_index/archive/milvus_update.zip
</pre><p>What changes are contained on this branch from Filip? Are there plans to merge this branch into main?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/1158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Yes, but not before the workshop</p>